app/assets/stylesheets/pages/_home.scss

```diff
@@ -1,14 +1,13 @@
 // Specific CSS for your home-page
 body{
     background-color: #F6F6F3;
+    padding-top: 70px;
 }
-.row{
-    text-align: center;
-    margin-top: 6rem!important;
```

Comment:
@vlopergolo was there any special reason why you added this margin-top on the row class? it was conflicting with the row class from bootstrap used on the /hikes page adding too much margin top on the begining